15-lhcb-dev.md

```diff
@@ -76,16 +76,41 @@ It works similar to `lb-run`, without the need to specify a package and version:
 
 > ## What if getpack asks for my password 1000 times? {.callout}
 > `getpack` might ask you for your password several times.
-> To avoid this, you can create a kerberos token with
-> ```
-> kinit
-> ```
-> You will have to enter your password once, and further password prompts will be skipped
+> To avoid this, you need to configure your `ssh` following the
+> instructions [here](http://information-technology.web.cern.ch/book/how-start-working-svn/accessing-svn-repository#accessing-sshlinux).
+> The summary of the instructions follows:
```

Comment:
I would add a bit of explanation along the lines of:
Since your home directoy on lxplus is on afs, you need a valid afs token to access it. Such a token can only be obtained when logging in with password or kerberos based authentication, and not when using public keys.

Since an afs token is not needed when communicating with the svn server, public key based authentication can be used there.